devel/py-backports.zoneinfo: New port

yzgyyang opened the pull request:
Backport of the standard library module zoneinfo.

PR:		256883
Submitted by:	rozhuk.im@gmail.com
Approved by:	lwhsu (mentor)

lwhsu reviewed (approved):
I know that you're taking the maintaiership, but I think it's fine to keep the submitter as patch author?

rozhuk-im committed:
devel/py-backports.zoneinfo: New port

Backport of the standard library module zoneinfo.

PR:		256883
Approved by:	lwhsu (mentor)
